game.js

```diff
@@ -36,6 +22,75 @@ const saveStates = (playerState, savedState) => {
   });
 };
 
+const createBoard = (size) => {
+const gameContainer =  document.querySelector('.game');
+gameContainer.classList.remove("hidden");
+if(game.boardSize !== "")
+gameContainer.classList.remove(game.boardSize);
+gameContainer.innerHTML = "";
+for(let i = 0; i < (size**2); i++){
+  const cell = document.createElement('div');
+  cell.classList.add(`grid-cell`);
+  cell.dataset.value = i;
+  gameContainer.appendChild(cell);
+  game.boardSize =`x${size}`
+}
+gameContainer.classList.add(game.boardSize);
```

Comment:
why not use `size` param itself?

Reply:
beacause I use it in another function when I want to remove the class

Reply:
jot you. so just save it to variable (e.g. const boardSizeClass = `x${size}`) , and the use it when you need it inside this scope

Reply:
you mean to make it global?

Reply:
no. only at the very beginning of `createBoard` function